README.md

```diff
@@ -37,13 +37,15 @@ e.g Enable offline form submission with following settings for the forms availab
 
 ## Integration with izooto plugin.
 
+- The [iZooto](https://wordpress.org/plugins/izooto-web-push/) plugin must be installed and activated to use the izooto notification services.
```

Comment:
This has to be added to - https://github.com/rtCamp/pwa-extension#requirements

add in the bracket that push notifications are dependent on that.

Reply:
Yes, it makes sense to keep it in both places.